core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,34 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    # This phase is used to give time to Cromwell to abort all workflows upon shutdown. It's only used if system.abort-jobs-on-terminate = true
+    abort-all-workflows {
+      # /!\ This depends-on field should NOT be modified
+      depends-on = [service-unbind]
```

Comment:
One quick example - if you remove the defined dispatchers you're going to have a bad time.

I see that argument but this is also a config file for us as well. If a developer wants to tweak something, the obvious place to do it is the config file. Now it is who knows where.

My stance on the user side of the config file is they should only ever be using a file that only contains the specific overrides that they want, and those overrides should have been explicitly pointed out to them in the docs as something they might want to change.

But this is also why the `akka` block should be in `application.conf` and not `reference.conf`, these aren't settings that we're providing to the outside world, they're settings that *we* are using.